plugins/in_kafka/in_kafka.c

```diff
@@ -180,8 +180,11 @@ static int in_kafka_collect(struct flb_input_instance *ins,
 
         rd_kafka_message_destroy(rkm);
 
-        /* TO-DO: commit the record based on `ret` */
-        rd_kafka_commit(ctx->kafka.rk, NULL, 0);
+        
+        if(!ctx->enable_auto_commit) {
```

Comment:
Current behaviour is to have auto-commit false but the default below is now true which means we are not maintaining the old behaviour by default.